src/runtime/less/ts-icons.less

```diff
@@ -372,198 +373,186 @@ i {
+    top: -1px;
+    left: 1px;
+  }
+  .ts-icon-checked-alt:before {
+    left: 1px;
+  }
+  .ts-icon-remove:before, .ts-icon-more:before {
+    left: 0.5px;
+  }
+  .ts-icon-warning:before, .ts-icon-error:before {
+    left: 1px;
+  }
+  .ts-icon-settings:before {
+    top: 0.5px;
+    left: 0.5px;
+  }
+  .ts-icon-usersettings:before {
+    top: -1px;
+    left: 0.5px;
+  }
 }
 
 // Firefox .....................................................................
 
 .ts-engine-gecko {
-	.ts-icon-close:before {
-		top: -1.75px;
-		left: 0.85px;
-	}
-	.ts-icon-remove:before {
-		top: 1px;
-	}
-	.ts-icon-menuswitch:before {
-		top: 1px;
-	}
-	.ts-icon-addfield:before {
-		left: 0.25px;
-		top: 0.25px;
-	}
-	.ts-icon-addfilter:before {
-		top: 1px;
-	}
-	.ts-icon-appactivate:before {
-		left: 1px;
-		top: -1px;
-	}
-	.ts-icon-companysize:before,
-	.ts-icon-users:before {
-		left: 2px;
-	}
-	.ts-icon-checkbox:before,
-	.ts-icon-checkboxon:before {
-		top: -2px;
-		left: 1px;
-	}
-	.ts-icon-createdocument:before {
-		left: 0.75px;
-		top: -0.75px;
-	}
-	.ts-icon-checked-alt:before {
-		left: 0.75px;
-	}
-	.ts-icon-download:before {
-		left: 1px;
-	}
-	.ts-icon-drafts:before {
-		left: 0.1px;
-		top: -0.5px;
-	}
-	.ts-icon-rating:before,
-	.ts-icon-followed:before,
-	.ts-icon-unfollowed:before {
-		left: 1px;
-		top: -1.25px;
-	}
-	.ts-icon-remove:before,
-	.ts-icon-more:before {
-		top: -1px;
-		left: 0.5px;
-	}
-	.ts-icon-warning:before,
-	.ts-icon-error:before {
-		left: 1px;
-		top: -1px;
-	}
-	.ts-icon-usersettings:before {
-		top: -2px;
-		left: 0.25px;
-	}
+  .ts-icon-close:before {
+    top: -1.75px;
+    left: 0.85px;
+  }
+  .ts-icon-remove:before {
+    top: 1px;
+  }
+  .ts-icon-menuswitch:before {
+    top: 1px;
+  }
+  .ts-icon-addfield:before {
+    left: 0.25px;
+    top: 0.25px;
+  }
+  .ts-icon-addfilter:before {
+    top: 1px;
+  }
+  .ts-icon-appactivate:before {
+    left: 1px;
+    top: -1px;
+  }
+  .ts-icon-companysize:before, .ts-icon-users:before {
+    left: 2px;
+  }
+  .ts-icon-checkbox:before, .ts-icon-checkboxon:before {
+    top: -2px;
+    left: 1px;
+  }
+  .ts-icon-createdocument:before {
+    left: 0.75px;
+    top: -0.75px;
+  }
+  .ts-icon-checked-alt:before {
+    left: 0.75px;
+  }
+  .ts-icon-download:before {
+    left: 1px;
+  }
+  .ts-icon-drafts:before {
+    left: 0.1px;
+    top: -0.5px;
+  }
+  .ts-icon-rating:before, .ts-icon-followed:before, .ts-icon-unfollowed:before {
+    left: 1px;
+    top: -1.25px;
+  }
+  .ts-icon-remove:before, .ts-icon-more:before {
+    top: -1px;
+    left: 0.5px;
+  }
+  .ts-icon-warning:before, .ts-icon-error:before {
+    left: 1px;
+    top: -1px;
+  }
+  .ts-icon-usersettings:before {
+    top: -2px;
+    left: 0.25px;
+  }
 }
 
 // Exploder ....................................................................
 
 .ts-engine-explorer {
-	.ts-icon-close:before {
-		top: -2px;
-	}
+  .ts-icon-close:before {
+    top: -2px;
+  }
```

Comment:
@bomouridsen All indent has changed with this commit, so it's hard to merge into my branch. I may have overlooked something :cry: Make sure to uninstall those Prettier and Beautify plugins and also adjust your general settings to use tabs instead of spaces.